README.md

```diff
@@ -116,7 +124,9 @@ body, that says "hello".
 
 Your Answer:
 ```js
-// your code here
+$('#greeting').bind('click', function(){
```

Comment:
nice! We prefer `.on("click")`. Check out http://stackoverflow.com/a/11847066/850825